src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static final int MAX_VALUE = 100;
+
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        ColorSupplier randomColor = new ColorSupplier();
+        ball.setColor(randomColor.getRandomColor());
+        ball.setNumber(new Random().nextInt(MAX_VALUE + 1));
+        return ball;
+    }

```

Comment:
We can add fields with type `Random` and `ColorSupplier`  to our class, so we won't need to create an instance every time when method is invoked
Also, we could choose better name for variable with type `ColorSupplier` (like `colorSupplier`)